app/views/courses/gcse_requirements/edit.html.erb

```diff
@@ -0,0 +1,45 @@
+<%= content_for :page_title, title_with_error_prefix("GCSEs and equivalency tests", @errors && @errors.any?) %>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link_to provider_recruitment_cycle_course_path %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">
+      <span class="govuk-caption-l"><%= course.name_and_code %></span>
+      GCSEs and equivalency tests
+    </h1>
+
+    <%= form_with(
+      model: @gcse_requirements_form, url:
+      gcses_pending_or_equivalency_tests_provider_recruitment_cycle_course_path(
+        @course.provider.provider_code,
+        @course.provider.recruitment_cycle_year,
+        @course.course_code,
+      ),
+      method: :put,
+      builder: GOVUKDesignSystemFormBuilder::FormBuilder,
+    ) do |form| %>
```

Comment:
Add error summary above page heading:

```suggestion
    <%= form_with(
      model: @gcse_requirements_form, url:
      gcses_pending_or_equivalency_tests_provider_recruitment_cycle_course_path(
        @course.provider.provider_code,
        @course.provider.recruitment_cycle_year,
        @course.course_code,
      ),
      method: :put,
      builder: GOVUKDesignSystemFormBuilder::FormBuilder,
    ) do |form| %>
   
    <%= form.govuk_error_summary %>
      
    <h1 class="govuk-heading-l">
      <span class="govuk-caption-l"><%= course.name_and_code %></span>
      GCSEs and equivalency tests
    </h1>
```

Reply:
Can we also use `f` rather than `form` (trying to ensure GOV.UK form builder forms use `f`, while older forms use `form`, to make it clearer which are using which builder)